### DISCUSSION TITLE
Pacstall

### DISCUSSION THREAD

**Discussion by trymeouteh** [Eloston/ungoogled-chromium#1925] [Ideas]
Please add this app into the pacstall repository. Pacstall is the AUR for Ubuntu which will allow Ubuntu users to easily install the app and receive updates very shortly after an update is released.

https://pacstall.dev/

**Comment by networkException** [comment]
This repository is not responsible for the packaging on certain platforms. Please open an issue in https://github.com/ungoogled-software/ungoogled-chromium-debian instead.
